src/django/api/admin.py

```diff
@@ -1,4 +1,15 @@
 from django.contrib import admin
-from .models import ThirdPlaceUser
+from .models import Location, ThirdPlaceUser
 
 admin.site.register(ThirdPlaceUser)
```

Comment:
had to register the BusinessHours and LocationType models here:
```py
admin.site.register(BusinessHours)
admin.site.register(LocationType)
```
in order to add items to the corresponding tables and then select from these in the "Add location" screen. the `hours` field will need to be converted to an array of `BusinessHours` (ManyToManyField?) – or there could be a column for each day of the week (`mon_hours`, `tues_hours`, etc).

anyway, this works!